### PR TITLE
Development

### PR DESCRIPTION
**Pull request by davidvenuto**
Added caching to improve CPU performance

### CODE REVIEW

**Review by davidvenuto — COMMENTED** [review]
Wow, this really is a stupendous way to generate multiple data sets. Good job boo, I love the way you summarize things on line 3 of your optimization document

File: src/main/java/Implementations/ComputationImpl.java [hunk 1]
```diff
@@ -40,27 +48,20 @@ public long[][] performDigitFactorial(String[] numberStrings) {
...
 
-    private static long factorial(int n) {
-        long result = 1;
-        for (int i = 2; i <= n; i++) {
-            result *= i;
-        }
-        return result;
-    }
-
     private static long digitFactorialSum(String numberString) {
```
Comment (Lumbus22) [inline src/main/java/Implementations/ComputationImpl.java:59]: Simple and effective implementation of using the array to perform a factorial on number 0-9.

**Review by Lumbus22 — COMMENTED** [review]
LGTM

**Review by Lumbus22 — APPROVED** [review]
LGTM

**Pull request merged**